website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,285 @@
+
+- the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) to display the tooltip content as "popover" on top of the page
+- A third-party library called [Floating UI](https://floating-ui.com/) provides the anchoring of the "popover" to the "toggle" element and the automatic positioning/collision detection functionality.
+- These functionalities are abstracted into two HDS utilities (the `hds-anchored-position` modifier and the [PopoverPrimitive component](/utilities/popover-primitive)), which the Rich Tooltip component is built on top of.
+
+Since tooltips are notoriously hard to implement in an accessible way, we limited the ways in which the Rich Tooltip can be used. At the same time, we have provided an escape hatch if some edge cases need to be supported (but in this case, consumers will need to make sure the component is used in a [conformant accessible way](/components/rich-tooltip?tab=accessibility)).
+
+The component provides a lot of options to customize its behavior. Below we have tried to describe the most common examples, but if you find a use case that is not supported by the existing Rich Tooltip implementation, please [speak with the Design System team](/about/support).
+
+## How to use this component
+
+When using this component, there are a few things to consider:
+
+- what toggle to use; the [standard toggle](#standard-toggle) or a [generic toggle](#generic-toggle)
+- if the toggle is a [standalone element](#as-a-standalone-element) or if instead lives [inline with other text](#inline-with-other-text)
+- how the end-user should [interact](#interactivity) with the toggle to show/hide the tooltip
+
+Depending of these factors, there are different ways to implement the code, described in these alternatives below.
+
+### Standard toggle
+
+The standard `Toggle` element **ensures perceivability and accessibility out of the box**. It consist of a text (with a specific underline decoration applied to it) and an optional icon. The icon can be leading or trailing, and its size is always proportional to the font size of the text (`1em`).
+
+The text and the icon are rendered inside an HTML `<button>` element (the one that technically acts as a toggling control for the popover). For details about how the user can interact with this button see the [Interactivity](#interactivity) sub-section.
+
+
+The `Bubble` element instead is a pure container that yields the children inside the "popover" tooltip bubble. Consumers can pass whatever content they need to it, but they are also responsible of styling and structuring it according to their needs/context.
+
+
+!!! Info
+
+Notice: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support).
```

Comment:
```suggestion
Note: we apply a CSS reset (`all: initial`) to the container to avoid styles applied to the parent elements leaking into the tooltip content. If you find any issue with this reset, please [speak with the Design System team](/about/support).
```